# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: C:\Zend\framework\documentation\manual\en\module_specs
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Zend_Tool-Usage-CLI.xml
--- Zend_Tool-Usage-CLI.xml Base (BASE)
+++ Zend_Tool-Usage-CLI.xml Locally Modified (Based On LOCAL)
@@ -47,7 +47,7 @@
             </para>
 
             <programlisting language="text"><![CDATA[
-pear discover-channel pear.zfcampus.org
+pear channel-discover pear.zfcampus.org
 pear install zfcampus/zf
 ]]></programlisting>
 
